docs/eventing/sugar/README.md

```diff
@@ -38,16 +35,33 @@ the default settings:
 
 There might be cases where automated Broker creation is desirable, such as on
 namespace creation, or on Trigger creation. The Sugar controller enables those
-use-cases:
+use-cases. The following sample configuration of the `sugar-config` ConfigMap
+enables Sugar Controller for all Namespaces & Triggers.
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+    name: config-sugar
+    namespace: knative-eventing
+    labels:
+        eventing.knative.dev/release: devel
+    data:
+      # Use an empty object to enable for all namespaces
+      namespace-selector: |
+        {}
+
+      # Use an empty object to enable for all triggers
+      trigger-selector: |
+        {}
+    ```
 
```

Comment:
In this case don't ident
```suggestion
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
name: config-sugar
namespace: knative-eventing
labels:
    eventing.knative.dev/release: devel
data:
  # Use an empty object to enable for all namespaces
  namespace-selector: |
    {}

  # Use an empty object to enable for all triggers
  trigger-selector: |
    {}
```

```